pages/header.py

```diff
@@ -1,25 +1,92 @@
-from methods import Methods
 from locators import LocatorsHeader
+from locators import LocatorsCreateCompanyPage
+from locators import LocatorsMyCompaniesPage
+from locators import LocatorsYourResume
 
 
-class Header(Methods):
+class Header:
     """Header page, which will be inherited by other pages"""
-    def __init__(self, driver):
-        super().__init__(driver)
-        self.driver = driver
+    def  __init__(self, browser):
+        self.browser = browser
         self.locators = LocatorsHeader
+        self.locator_create_company = LocatorsCreateCompanyPage
+        self.locator_my_companies = LocatorsMyCompaniesPage
+        self.locators_your_resume = LocatorsYourResume
 
     def click_icon(self):
         """Clicks on the round icon"""
-        self.click_element(self.locators.ICON)
+        self.browser.click_element(self.locators.ICON)
+
+    def check_dropdown(self):
+        """Verifies if the dropdown box appeared"""
+        flag = self.browser.get_attr_value(self.locators.CHECK_DROPDOWN, 'aria-expanded')
+        return True if flag == 'true' else False
+
+    def click_log_out(self):
+        """Clicks on the round icon"""
+        self.browser.click_element_by_text_simple(self.locators.LOG_OUT)
+
+    def click_log_in(self):
+        """Clicks on the round icon"""
+        self.browser.click_element_by_text_simple(self.locators.LOG_IN)
+
+    def is_logined(self):
+        """ Check if user logined: if 'logout' button exist == logined """
+        self.click_icon()
+        log_out = self.browser.get_elements_by_text(self.locators.LOG_OUT)
+        return log_out.text == 'Log out'
 
     def select_option(self, pick_item):
         """The pick_item is default string parameter which accepts only 'Log in', 'Profile', 'Log out' values"""
         values = ('Log in', 'Profile', 'Log out')
         if pick_item not in values:
             raise Exception('Incorrect value for click_dropdown function')
-        self.click_element_by_text(self.locators.DROPDOWN, pick_item)
 
-    def transit(self, pick_item):
-        self.click_icon()
-        self.select_option(pick_item)
\ No newline at end of file
+       # if self.check_dropdown():
+        url_0 = self.browser.driver.current_url
+        while True:
+            self.click_icon()
+            print('clicked icon')
+            while True:
+                if self.check_dropdown():
+                    print('dropdown available')
```

Comment:
did you forget to remove `print` or you decided left it there? if yes, please, explain why

Reply:
the plan is to check every step